src/settings.ts

```diff
@@ -49,6 +50,7 @@ export const SPARKLE_ICON = "/sparkle-icon.png";
 export const DEFAULT_MAP_BACKGROUND = "/maps/Sparkle_Field_Background.jpg";
 export const DEFAULT_VENUE_BANNER = "/assets/Sparkle_Banner_Default.jpg";
 export const DEFAULT_VENUE_LOGO = "/assets/Sparkle_SquareLogo_Default.jpg";
+export const SSO_LOGIN_ICON = "/assets/SSO_Login_Logo.png";
```

Comment:
See comments elsewhere about importing this icon from fontawesome or similar.

If we do choose to keep doing it this way, then it shouldn't be loaded as a magic string like this, but should moved to somewhere in `src/assets/*`, and then loaded via `import` so that it is properly bundled/checked/etc.

Reply:
> See comments elsewhere about importing this icon from fontawesome or similar.

There were no needed icon

> If we do choose to keep doing it this way, then it shouldn't be loaded as a magic string like this, but should moved to somewhere in src/assets/*, and then loaded via import so that it is properly bundled/checked/etc.


Done in https://github.com/sparkletown/sparkle/pull/1546/commits/c5f8848eb58b5921d9ff5785eba01ee6b0bb2278

Reply:
> There were no needed icon

😕 I showed a bunch of icons that would have worked equally as well for this in those other comments. And from the design mockup in https://github.com/sparkletown/internal-sparkle-issues/issues/767#issuecomment-861814262 it looks like using the GitHub logo would be appropriate)

Reply:
> I showed a bunch of icons that would have worked equally as well for this in those other comments. And from the design mockup in sparkletown/internal-sparkle-issues#767 (comment) it looks like using the GitHub logo would be appropriate)

This is a placeholder icon, I'll remove it as soon as we have final designs

Reply:
If it’s a placeholder then even more reason to use something from the icon line and not add yet another file to forever clutter and bloat our git history.